src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -99,23 +102,11 @@ protected RemoteConfigTemplate execute() throws FirebaseRemoteConfigException {
 
   @VisibleForTesting
   FirebaseRemoteConfigClient getRemoteConfigClient() {
-    return remoteConfigClient.get();
+    return remoteConfigClient;
   }
 
   private static FirebaseRemoteConfig fromApp(final FirebaseApp app) {
-    return FirebaseRemoteConfig.builder()
-            .setFirebaseApp(app)
-            .setRemoteConfigClient(new Supplier<FirebaseRemoteConfigClient>() {
-              @Override
-              public FirebaseRemoteConfigClient get() {
-                return FirebaseRemoteConfigClientImpl.fromApp(app);
-              }
-            })
-            .build();
-  }
-
-  static Builder builder() {
-    return new Builder();
+    return new FirebaseRemoteConfig(app);
```

Comment:
Remove `fromApp`. Call constructor directly where it's needed.

Reply:
Thanks! Updated!